src/tests/ftest/rebuild/no_cap.py

```diff
@@ -156,7 +156,7 @@ def test_rebuild_no_capacity(self):
         pool_checks["pi_ndisabled"] = ">0"
         rebuild_checks["rs_obj_nr"] = ">=0"
         rebuild_checks["rs_rec_nr"] = ">=0"
-        rebuild_checks["rs_state"] = 0
+        rebuild_checks["rs_state"] = ">=0"
```

Comment:
Is it possible for `rs_state` to be something other than 0 or 1? I.e., @wangdi1 mentions 0 or 1 is valid: https://daosio.atlassian.net/browse/DAOS-10217
But is is possible for query to return, for example, `rs_state == 2`?

Reply:
I think rs_state will not return 2.

Reply:
Thanks. LGTM